.htmllintrc

```diff
@@ -0,0 +1,70 @@
+{
+  "attr-bans": [
+    "align",
+    "background",
+    "bgcolor",
+    "border",
+    "dynsrc",
+    "id",
```

Comment:
I think that `id` can be "authorized" to preserve same behaviours of our old code base